compiler/backend_inkwell/Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "backend_inkwell"
```

Our packages start with `candy_`:

```suggestion
name = "candy_backend_inkwell"
```